packages/amplication-client/src/Workspaces/hooks/useResources.ts

```diff
@@ -91,8 +93,7 @@ const useResources = (
 
   const createService = (
     data: models.ResourceCreateWithEntitiesInput,
-    eventName: AnalyticsEventNames,
-    addEntity: (id: string) => void
+    eventName: AnalyticsEventNames
```

Comment:
Since you removed the third parameter, I expect to see a change in `packages/amplication-client/src/Resource/create-resource/CreateServiceWizard.tsx`